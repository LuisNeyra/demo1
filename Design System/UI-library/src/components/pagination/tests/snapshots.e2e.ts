import { snapIt } from '../../../test';

const component = 'pagination';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('should work without parameters', '<blaze-pagination></blaze-pagination>');
    snap('renders the correct buttons when pages set', '<blaze-pagination pages="15"></blaze-pagination>');
  });
});
