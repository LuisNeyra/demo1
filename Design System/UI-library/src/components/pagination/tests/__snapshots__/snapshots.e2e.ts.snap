// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pagination renders renders the correct buttons when pages set 1`] = `"<blaze-pagination pages=\\"15\\" class=\\"hydrated\\"><nav class=\\"c-pagination\\"><button class=\\"c-pagination__control\\" disabled=\\"\\">‹</button><div class=\\"c-pagination__pages\\"><button class=\\"c-pagination__control\\" aria-current=\\"\\">1</button><button class=\\"c-pagination__control\\">2</button></div><button class=\\"c-pagination__control\\">›</button></nav></blaze-pagination>"`;

exports[`pagination renders should work without parameters 1`] = `"<blaze-pagination class=\\"hydrated\\"><nav class=\\"c-pagination\\"><button class=\\"c-pagination__control\\" disabled=\\"\\">‹</button><div class=\\"c-pagination__pages\\"><button class=\\"c-pagination__control\\" aria-current=\\"\\">1</button></div><button class=\\"c-pagination__control\\" disabled=\\"\\">›</button></nav></blaze-pagination>"`;
